src/components/Copyright/Copyright.tsx

```diff
@@ -159,6 +159,17 @@ const License: React.FC = () => {
         , you may copy, distribute and modify the work as long as any
         modifications are also made available under the GPL.
       </p>
+      <Heading level="h2">In Copyright</Heading>
+      <p>
+        Works that are{" "}
+        <Link to="https://rightsstatements.org/page/InC/1.0/?language=en">
+          In Copyright
+        </Link>{" "}
+        are protected by copyright and/or related rights. You are free to use
+        this Item in any way that is permitted by the copyright and related
+        rights legislation that applies to your use. For other uses you need to
+        obtain permission from the rights-holder(s).
+      </p>
```

Comment:
Is there an updated BRD? I see that this is the language reflected in the ticket but not here: https://docs.google.com/document/d/12WVwIxC_OT3eWcw8-LQd1U-FX91tj-qWnL8A6d0CLsU/edit

Also https://rightsstatements.org/vocab/InC/1.0/ unfortunately is returning a Bad Gateway. I'd let Will know and have him make decision on whether we want to include it or not.